apiserver/facades/client/resources/repository.go

```diff
@@ -15,16 +15,3 @@ import (
 type NewCharmRepository interface {
```

Comment:
Might as well delete this file and move this interface into the facade.

Reply:
This file shouldn't be deleted - we need the localClient.ResolveResources method. Not all of the AddPendingResources code has been replaced yet.